scripts/run.sh

```diff
@@ -21,5 +21,12 @@ fi
 echo "Starting service..."
 docker create -v /etc/nginx/ssl --name $SERVICEVOLUMENAME ubuntu
 docker cp $SSL_PATH/. $SERVICEVOLUMENAME:/etc/nginx/ssl/.
-docker run -h $CONTAINERHOSTNAME --restart unless-stopped -t -d -p 8080:8080 --name $SERVICENAME --volumes-from $SERVICEVOLUMENAME -e APPINSIGHTS_KEY=$APPINSIGHTS_KEY -e CONTAINER_VERSION=$CONTAINERTAG -e affinity:image==$CONTAINERNAME $CONTAINERNAME
+
+if docker container inspect -f . $CREDSCANSERVICEVOLUMENAME; then
+  echo "Running with Credential Scanner"
```

Comment:
CredScanner will be an important component moving forward.  Can we add a pointer in the readme to a brief write-up?